src/sentry/utils/email.py

```diff
@@ -389,3 +392,29 @@ def send_mail(subject, message, from_email, recipient_list, fail_silently=False)
         subject, message, from_email, recipient_list,
         connection=get_connection(fail_silently=fail_silently),
     )
+
+
+class PreviewBackend(BaseEmailBackend):
+    """
+    Email backend that can be used in local development to open messages in the
+    local mail client as they are sent.
+
+    Probably only works on OS X.
+    """
+    def send_messages(self, email_messages):
+        for message in email_messages:
+            content = str(message.message())
+            preview = tempfile.NamedTemporaryFile(
+                delete=False,
+                prefix='preview-',
+                suffix='.eml',
+            )
+            preview.write(content)
+            preview.flush()
+            preview.close()
+            try:
+                subprocess.check_call(('open', preview.name))
```

Comment:
Damn, I actually had this leaving it on the filesystem earlier (see in getsentry) but I added it here after a quick test with Mail.app. I'll remove it here, I guess. :disappointed: